internal/driver/constants.go

```diff
@@ -76,4 +80,17 @@ const (
 	UdevSerialShort = "ID_SERIAL_SHORT"
 	UdevSerial      = "ID_SERIAL"
 	UdevV4lProduct  = "ID_V4L_PRODUCT"
+
+	// unsupported pixel formats
+	PixelFmtDepth     = 540422490
+	PixelFmtDepthDesc = "16-bit depth"
+
+	PixelFmtUYVY     = 1498831189
+	PixelFmtUYVYDesc = "UYVY 4:2:2"
```

Comment:
You do not need this block. V4l2 library supports this. You can look into my PR if needed for reference.